setup/install_upstream_rust_keylime/test.sh

```diff
@@ -57,6 +74,15 @@ _EOF'
 [Service]
 Environment=\"RUST_LOG=keylime_agent=trace\"
 _EOF"
+            if [ -z "$KEYLIME_RUST_CODE_COVERAGE" ]; then
+                rlRun "cat > /etc/systemd/system/keylime_agent.service.d/10-coverage.conf <<_EOF
```

Comment:
I am thinking we should maybe rename the file to `15-rust-coverage.conf` to avoid conflict with the one created for Python. It should not do any harm if both files would be present, however with conflicting names in place the order of setup script execution would matter.